pyodide_build/recipe/skeleton.py

```diff
@@ -320,3 +320,63 @@ def update_package(
     run_prettier(meta_path)
 
     logger.success(f"Updated {package} from {local_ver} to {pypi_ver}.")
+
+
+def disable_package(recipe_dir: Path, package: str, message: str) -> None:
+    yaml = YAML()
+
+    meta_path = recipe_dir / package / "meta.yaml"
+    if not meta_path.exists():
+        logger.error("%s does not exist", meta_path)
+        raise MkpkgFailedException(f"{package} recipe not found at {meta_path}")
+
+    yaml_content = yaml.load(meta_path.read_bytes())
+    pkg = yaml_content["package"]
+    pkg_keys = list(pkg)
+    # Insert after the version key
+    version_idx = pkg_keys.index("version") + 1
+    pkg.insert(version_idx, "_disabled", True)
+    # Add message above it
+    if message:
+        pkg.yaml_set_comment_before_after_key("_disabled", before=message)
+    yaml.dump(yaml_content, meta_path)
+    run_prettier(meta_path)
```

Comment:
Maybe let's put the `npx` check into the `run_prettier` function, as `run_prettier` is now used in multiple places.

https://github.com/pyodide/pyodide-build/blob/7b43b6730014208ed37f564f37bb581fad24d8a7/pyodide_build/recipe/skeleton.py#L206-L211